py/torch_tensorrt/_features.py

```diff
@@ -62,6 +64,22 @@ def not_implemented(*args: List[Any], **kwargs: Dict[str, Any]) -> Any:
     return wrapper
 
 
+def needs_refit(f: Callable[..., Any]) -> Callable[..., Any]:
```

Comment:
Maybe we can make this a bit more generic (like for any feature in the FeatureSet)